### PR TITLE
edits

### PR DESCRIPTION
**Pull request by krishahn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,145 @@
 # The API Console
 
-MuleSoft's API Console is a fully-fledged API documentation tool that generates mobile-friendly web-documentation based on RAML (Restful API Modeling Language) documents. It also provides a mechanism to let consumer directly interact with an API.
+MuleSoft's API Console is a full-fledged API documentation tool that generates mobile-friendly web documentation based on RAML (Restful API Modeling Language) documents. A mechanism provides direct interaction between the consumer and an API.
```
Comment (sichvoge) [inline README.md]: Hi Kris, `A mechanism provides ...` doesn't seem to be right here. It should be a follow-up sentence that additionally to just providing documentation, it also gives users to ability to try it out requests on the fly.

File: README.md [hunk 2]
```diff
@@ -1,134 +1,145 @@
...
 
 To build the API Console as a standalone application use one of our [build tools].
 
 ### Embed as an HTML element
 
-The API Console was build on top of new Web Components specification. When you include sources of the console into your web application it will register a new HTML element, the `<api-console>` element, that can be used as any other element on the page or web application. This means that you can embed the console into your blog post or as a part of a press release, and your users will be able to get know your API without being redirected to another web page.
+The API Console was built on top of a new Web Components specification. When you include sources of the console into your web application it registers a new HTML element, `<api-console>`. You can use this element in the same way as any other element on the page or web application. For example, you can embed the console into your blog post or as a part of a press release. Your users can explore your API without being redirected to another web page.
```
Comment (sichvoge) [inline README.md]: Instead of `on top of a new Web Components specification` it should be `on top of the new Web Components specification`, right?

Comment (krishahn) [inline README.md]: fixed

**Comment by jarrodek** [comment]
Thanks @sichvoge

**Pull request merged**